configs/layer-config.ts

```diff
@@ -1 +1,8 @@
 export const allowedLayers = ['feature', 'dynamic', 'loss', 'gain']; //To be: tiled, webtiled, image, dynamic, feature, graphic, and custom (loss, gain, glad, etc)
+
+//Layer controls (IDS)
+export const densityEnabledLayers = [
+  'TREE_COVER_LOSS',
+  'AG_BIOMASS',
+  'TREE_COVER'
+];
```

Comment:
@vaidotasp Looks like the country config files might be tracking densityLayers via the property `GROUP_LC`. Just flagging in case we need to revisit this later

Reply:
which property is tracking this in country config? I was looking through that but it was unclear to me which layers have this ability and which do not 🤔

Reply:
Looks like `layerPanel[GROUP_LCD]` is tracking density (and I suspect a few other things). I noticed the function `updateDensity()` from `v1` leverages that object within the country config to maintain density, so I wanted to flag it for a future PR or ticket

```javascript
// LossControls.js, v1
  updateDensity (layer, density) {
    const { settings } = this.context;
    const layerGroups = settings.layerPanel;
    const layerConf = utils.getObject(layerGroups.GROUP_LCD.layers, 'id', this.props.layerId);
    let baseUrl = layerConf.url;
    baseUrl = baseUrl.split('tc')[0] + 'tc';
    baseUrl += density;
    baseUrl += '/{z}/{x}/{y}.png';    

    if (layer && layer.setUrl) {
      layer.setUrl(baseUrl);
    }
  }
```